js/tji-datasets-explore.js

```diff
@@ -206,6 +208,7 @@ var TJIChartView = function(chart_configs, charts_elt_id, filters_elt_id, chart_
   this.chart_wrapper = chart_wrapper;
   this.count_template = count_template;
   this.filters = null;
+  this.state.$loader = jQuery(loader_html).prependTo(this.charts_elt_id);
```

Comment:
see below comment about including the loader on page load. This way the user gets the loader immediately, and isn't waiting for js to load to see it.

Reply:
seenzo donezo